### ISSUE TITLE
Microsoft.Data.SqlClient.SqlException : 'Could not find stored procedure 'dbo.p_SetupWindowsNetworkUser'.'

### ISSUE THREAD

**Issue by kilroyFR** [LSNicholls/Serenity-WinAuthSample#1]
hello there, thanks for sharing !

i wanted to try and rebuild project but got this exception when running. 
thanks !

**Comment by LSNicholls** [comment]
Yes, that is in the instructions/readme!  At least I think it is... let me check... yep... screenshot below.

FWIW the version of sqllocaldb you have on your machine is based on the first version of Visual Studio you installed on that machine, it seems to me.  It doesn't automatically upgrade later.  However [that article I linked](https://intellitect.com/blog/upgrading-sql-server-localdb/) to worked great for me on the machine where I found that problem.  In my installation, I had multiple instances when I ran the "info" command -- so I followed the instructions to delete both of them and just recreated the default (MSSQLLocalDB) one.

If this does *not* work for you, then you don't have a recent version of SQL Server available on your dev machine to help upgrade sqllocaldb!  In that case, I will publish the very minor changes that you will have to make in the procedure in the sql file to downgrade it.  You can also figure them out by just running the sql file manually and noting where the error occurs -- I think it is just the trim() function, which was introduced in 2017. <s>

![image](https://github.com/LSNicholls/Serenity-WinAuthSample/assets/88156108/2eeaa4eb-dd48-4041-89df-ef909da1bbb9)

**Comment by LSNicholls** [comment]
@kilroyFR I hope that worked for you - you should have been able to build and run after fixing mslocaldb.  I just want to say that I am going to wait to make any published changes to this project until the Serenity folks publish an updated (AspNet Core v8) VSIX for Serene, because I cobbled this together from a v6 one and it was, and remains, a nightmare.  Their premium template (Start Sharp), which I use internally, is already at v8 and works fine with my winauth stuff, but I know the public project based on Serene can easily break if you update Serenity components yourself, mostly having to do with their removing reliance on jQuery I think.
